src/docs/integration/linux/reference-designs/qemu-arm64/overview.md

```diff
@@ -0,0 +1,70 @@
+# Overview
+
+## Getting Started
+
+This guide depends on the following requirements:
+
+* [Peridio Cloud Configuration](/integration/linux/overview#peridio-cloud-requirements)
+* [Peridio CLI Development Machine](/integration/linux/overview#development-machine-requirements)
+
+### Configuration
+
+#### Generate Device Identity
+
+Export the environment variables `PERIDIO_CERTIFICATE_FILE` and `PERIDIO_PRIVATE_KEY_FILE` to the location of the pem files generated when configuring your signing PKI. More information about generating device signing PKI, reference the [x509 certificate guide](/platform/guides/creating-x509-certificates-with-openssl).
```

Comment:
> the location of the pem files

"pem" should be "PEM"

> More information about generating device signing PKI, reference the [x509 certificate guide](/platform/guides/creating-x509-certificates-with-openssl).

This sentence reads a bit incomplete and can be more in-line with wider docs if reworked to something like:

For more information about device signing PKI, see [creating CA certificates](/platform/guides/creating-x509-certificates-with-openssl).